Homework/Marianna/ClassesObjectsInheritance/ShapeClass/Pyramid.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ShapeClass
+{
+    class Pyramid : ThreeDimensionalShape
+    {
+        public Pyramid(double a, double sideheight, double pyramidHeight) : base(a)
+        {
+            Apophema = sideheight;
+            PyramidH = pyramidHeight;
+        }
+        public double Apophema { get; set; }
+        public double PyramidH { get; set; }
```

Comment:
`PyramidH` is a property defined inside a class already named `Pyramid`. Maybe just `Height` would be better?